osgar/drivers/test_kloubak.py

```diff
@@ -84,4 +84,21 @@ def test_invalid_can_message(self):
 
         k2.update_encoders(msg_id=0x92, data=b'\x97')
 
+    def test_partial_axis_encoders_update(self):
+        config = {}
+        bus = MagicMock()
+        k3 = RobotKloubak(config, bus)
+        self.assertIsNone(k3.partial_axis_encoders_update((121, 2, 2280, 2486)))
+
+        # epoch change, but incomplete data -> do not return anything
+        self.assertIsNone(k3.partial_axis_encoders_update((122, 1, 2183, 3786)))
+
+        # now proper update
+        self.assertIsNone(k3.partial_axis_encoders_update((122, 2, 2280, 2486)))
+#        self.assertIsNone(k3.partial_axis_encoders_update((122, 3, 2164, 2486)))
```

Comment:
why the tests are not working? Maybe `config` has to be modified from empty to define K2?